### DISCUSSION TITLE
这是朱明第一杀手，你敢和她对视十秒吗？

### DISCUSSION THREAD

**Discussion by 521888zX** [share121/inter-knot#485] [其他]
这是朱明第一杀手，你敢和她对视十秒吗？
![1721894047780](https://github.com/user-attachments/assets/96a46f73-9b82-400f-b3c9-a43d9a224005)
![Screenshot_20240730_121642](https://github.com/user-attachments/assets/714bb987-265a-46e0-8b0e-664b151e1d01)
![Screenshot_20240730_121753](https://github.com/user-attachments/assets/9aa8f31f-4089-494f-a57a-1c5e04fe8d5d)

**Comment by swyh51982** [comment]
天呐，这太可怕了(=°Д°=)

**Reply by YTYX-Github** [reply]
天呐 这太不可怕了

**Comment by gifttoword** [comment]
我家小孩ヾ(๑╹ヮ╹๑)ﾉ”

**Comment by warning-Kirov-reporting** [comment]
这是我家小孩

**Comment by lllliufeng** [comment]
如果你敢跟她对视超过十秒，说明你的冷血指数爆表

**Comment by KUNKUN4** [comment]
已经对视10秒了

**Comment by circle-P** [comment]
好可怕∑(;°Д°)

**Comment by XiaoFffeng** [comment]
谁家小孩

**Comment by ZL2007627** [comment]
这太可怕了。

**Comment by Mnpolize** [comment]
完了对视了，我不会寄了吧（哭

**Comment by ShuoChen08** [comment]
谁家小孩

**Comment by Frieezed** [comment]
谁家小孩

**Comment by 1155993355** [comment]
有一个黄头发的少年，他说有一个女孩拿着他的剑，外貌和这上面的很像，需要我赶走他不？不贵，1万菲林即可。

**Comment by Lie-groups-lie** [comment]
我家小孩！！！！

**Comment by hjejdjdj** [comment]
太可怕了

**Reply by clackwopck** [reply]
谁家小孩

**Comment by summer114514** [comment]
可爱晕了

**Comment by MissHedwig** [comment]
我家小孩

**Comment by hgfyurr** [comment]
真可爱

**Comment by FeiXiao-zm** [comment]
全体目光向我看齐嗷，我宣布个事  我是云璃的狗！！！！！！

**Reply by bilieh** [reply]
我才是

**Reply by FeiXiao-zm** [reply]
我才是我才是我才是！

**Reply by StarrySkyehk** [reply]
> 全体目光向我看齐嗷，我宣布个事 我是云璃的狗！！！！！！

你是逆天的

**Reply by FeiXiao-zm** [reply]
欸嘿

**Comment by ybj67** [comment]
↑独立宣言↑

**Reply by FeiXiao-zm** [reply]
哈哈哈哈哈哈哈哈哈哈哈哈哈哈

**Comment by SWYH134649** [comment]
我家小孩🥰

**Comment by glhongbo** [comment]
我老婆，欸嘿

**Comment by fengxia33** [comment]
把人可爱死的（确信）

**Comment by xiaomei-aihongwen** [comment]
对视一分钟，吓得射出来力

**Comment by dominic121-dot** [comment]
谁家小孩

**Comment by sky27950** [comment]
无敌剑士123：就我一个人对云璃无感吗？真不懂为什么这么多人喜欢她…

**Comment by Mayf1ower** [comment]
👆彦卿，何时来的

**Comment by nailbiting21** [comment]
天呐，实在是太可怕了

**Comment by Qiling-XwX** [comment]
瞪回去！
![64F9115DCF7D024CC23AB552B3B26142](https://github.com/user-attachments/assets/c95c1f13-f25a-44f7-9cbf-5ea962da34cc)

**Comment by aaa00073** [comment]
太可怕了，嘴角都吓哭了！

**Comment by Furina-D** [comment]
这是我家小孩

**Comment by Hanabiiiiii** [comment]
云璃小猫，你可以吃，琼实鸟串

**Comment by 123456wanh** [comment]
小时候看这集被防反了

**Comment by figs23314** [comment]
好可怕

**Comment by tanls930** [comment]
好恐怖啊啊啊

**Comment by T7moris** [comment]
确实很恐怖∑(;°Д°)
